charts/k8s-monitoring/templates/alloy-servicegraph.yaml

```diff
@@ -30,6 +30,9 @@
     {{- $alloyConfig = cat $alloyConfig ((include "servicegraph.connector.serviceGraphMetrics.alloy" $serviceGraphConfig) | trim | nindent 0) }}
 
     {{- range $dest := $destinations }} 
+      {{- if eq (include "secrets.usesKubernetesSecret" $dest) "true" }}
+        {{- $alloyConfig = cat $alloyConfig (include "secret.alloy" (deepCopy $ | merge (dict "object" $dest)) | nindent 0) }}
+      {{- end }}
       {{- $alloyConfig = cat $alloyConfig (printf "\n\n// Destination: %s (%s)" $dest.name $dest.type) }}
```

Comment:
Can we move the secret below this line that prints the `// Destination: name (type)`?